recipe/meta.yaml

```
@@ -30,36 +30,32 @@ requirements:
     - wheel
   run:
     - python
-    - appnope   # [osx]
-    - backcall
     - colorama  # [win]
     - decorator
     - exceptiongroup  # [py<311]
     - jedi >=0.16
     - matplotlib-inline
-    - pexpect >4.3  # [unix]
-    - pickleshare
-    - prompt-toolkit >3.0.30,<3.1.0,!=3.0.37
+    - pexpect >4.3
+    - prompt_toolkit >=3.0.41,<3.1.0
     - pygments >=2.4.0
     - stack_data
     - traitlets >=5
-    - typing_extensions  # [py<310]
 
 test:
   requires:
     - pip
     {% if not migrating %}
-    #- black
+    - pytest <7.1
+    - pytest-asyncio
```

```suggestion
    - pytest-asyncio <0.22
```
https://github.com/ipython/ipython/blob/8.20.0/setup.cfg#L75C19-L75C24